Framework/test/testQualitiesToFlagCollectionConverter.cxx

```diff
@@ -50,6 +50,10 @@ BOOST_AUTO_TEST_CASE(test_NoBeginning)
     { Quality::Bad, "xyzCheck", "DET", {}, {}, {}, { { metadata_keys::validFrom, "10" }, { metadata_keys::validUntil, "50" } } },
     { Quality::Good, "xyzCheck", "DET", {}, {}, {}, { { metadata_keys::validFrom, "50" }, { metadata_keys::validUntil, "120" } } }
```

Comment:
```suggestion
    { Quality::Bad, "xyzCheck", "DET" },
    { Quality::Good, "xyzCheck", "DET" }
```
please also remove the usage of metadata to set the validity, since it's not needed anymore and would only bring confusion.